app/views/posts/update_location.js.erb

```diff
@@ -0,0 +1,8 @@
+// note: $().html().text() allows for XSS but since this data comes from Rails it should be fine
+
+const decodedHtml = $('<div/>')
+  .html("<%= j (@posts.map { |post| render partial: 'postcard', locals: { post: post } }.join()) %>")
```

Comment:
need to add location distance to each post instead of showing just post lat long, but will do in later pr